src/parser/mod.rs

```diff
@@ -6809,6 +7000,46 @@ impl<'a> Parser<'a> {
                     trailing_bracket = _trailing_bracket;
                     Ok(DataType::Struct(field_defs))
                 }
+                Keyword::NULLABLE if dialect_of!(self is ClickHouseDialect) => {
```

Comment:
for the new entries where applicable, we can extend them to cover the GenericDialect as well (and update the test to cover accordingly)

Reply:
I updated all new entries with `GenericDialect`